### DISCUSSION TITLE
T-Deck with default settings (PowerSave On and Buzzer Enabled) freezes on message received

### DISCUSSION THREAD

**Discussion by jcyrio** [meshtastic/firmware#5237] [Troubleshooting]
When the T-Deck has Powersave mode on, and External Notifications enabled, and I2S buzzer enabled, it freezes after receiving a text message. It seems this might be related to the I2S board being connected to GPIO0 as an output, whereas GPIO0 is also used as the trackball button input for waking from light sleep.

Does anyone have any suggested fixes for this? Do you think this is something we might be able to tackle with software, or is it a hardware limitation?

It is unfortunate that the device has such a serious bug on the default settings. It gives new users a bad impression. If we're not able to fix the problem, maybe we could at least default to keeping Powersave mode off for this device?

**Comment by mverch67** [comment]
The audio issue has been fixed in a PR branch in commits
https://github.com/meshtastic/firmware/pull/3259/commits/8eb8c677e62794cea41f7ffcc715de0a5a4c7ae1, https://github.com/meshtastic/firmware/pull/3259/commits/3f02ee43cb38ebdf8345b75b6a19c561e92a7c67, and https://github.com/meshtastic/firmware/pull/3259/commits/86792f79407f3f30a0949caba2cbceea86b9bbd5, https://github.com/meshtastic/firmware/pull/3259/commits/9a9172ff521cd406fb62a3ff401c25f33cb582dd

**Reply by mverch67** [reply]
Audio issue fixed by https://github.com/meshtastic/firmware/pull/5422 in master
